src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Core/Services/ConnectionService.cs

```diff
@@ -39,19 +39,23 @@ public async Task<Result<IEnumerable<ConnectionDto>>> Get(Guid party, Guid? from
             return problem;
         }
 
-        if (party == from?.Id)
+        var connections = await connectionQuery.GetConnectionsAsync(
+        new ConnectionQueryFilter()
```

Comment:
Det må vel fortsatt gjøres to forskjellige oppslag (sett med filter parametere) her, basert på hvilken "rettning" man slår opp, gitt at:

ToOthers:
- Skal inkludere de som har nøkkelroller og klientdelegeringer som sub-connections

FromOthers:
- Skal inkludere underenheter som sub-connections

Er nå litt usikker på hvordan slutt resultatet av query løsningen til @Thuen ble. Du får der nå flat liste nå og at det er i Dto mappingen dette gjøres?